documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -59,26 +59,20 @@ metadata:
     createdForProviderType: <provider_type> <2>
 type: Opaque
 stringData:
-  user: <user> <3>
-  password: <password> <4>
-  domainName: <domain_name> <5>
-  projectName: <project_name> <6>
-  regionName: <region name> <7>
-  cacert: | <8>
-    <ca_certificate>
-  url: <api_end_point> <9>
-  thumbprint: <vcenter_fingerprint> <10>
+
+  user: <user> <1>
+  password: <password> <2>
+  insecureSkipVerify: <true/false> <3>
+  cacert: | <4>
+    <engine_ca_certificate>
+  thumbprint: <vcenter_fingerprint> <5>
+EOF
 ----
-<1> The `ownerReferences` section is optional.
-<2>  Specify the type of source provider. Allowed values are `ovirt`, `vsphere`, and `openstack`. This label is needed to verify the credentials are correct when the remote system is accessible and, for {rhv-short}, to retrieve the {manager} CA certificate when a third-party certificate is specified.
-<3> Specify the vCenter user, the {rhv-short} {manager} user, or the {osp} user.
-<4> Specify the user password.
-<5> {osp} only: Specify the domain name.
-<6> {osp} only: Specify the project name.
-<7> {osp} only: Specify the name of the {osp} region.
-<8> {rhv-short} and {osp} only: For {rhv-short}, enter the {manager} CA certificate unless it was replaced by a third-party certificate, in which case enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA. For {osp}, enter the CA certificate for connecting to the source environment.
-<9> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere, `https://<engine_host>/ovirt-engine/api/` for {rhv-short}, or `https://<identity_service>/v3` for {osp}.
-<10> VMware only: Specify the vCenter SHA-1 fingerprint.
+<1> Specify the vCenter user or the {rhv-short} {manager} user.
+<2> Specify the user password.
+<3> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration and that the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed. Specifying `<false>` verifies the certificate.
```

Comment:
not sure about this but should "that the certificate" change to "then the certificate"?
```suggestion
<3> Specify `<true>` to skip certificate verification, which proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed. Specifying `<false>` verifies the certificate.
```